Nancy.Rollbar/RollbarDataFactory.cs

```diff
@@ -78,8 +78,10 @@ public class RollbarDataFactory : IRollbarDataFactory {
                     .ToDictionary(x => x.Key, x => x.Value);
                 if (rr.PostParams.Count == 0) {
                     rr.PostParams = null;
-                }
-                rr.PostBody = context.Request.Body.AsString();
+                }
+                if (!context.Request.Body.CanRead) {
+                    rr.PostBody = context.Request.Body.AsString();

```

Comment:
The JsonSerializer won't send the key if PostBody is null. So there's actually a semantic difference between the two.